src/Search.js

```diff
@@ -10,15 +11,7 @@ const Search = () => {
         <div className="col">
           <form className="form-group search-box">
             <label htmlFor="customerName">Customer name</label>
```

Comment:
The `label` here is kind of attached to the `input` element that you moved into the component. At this point, it doesn't really matter whether the `input` is in the `SearchButton` component or the `Search` component (it may later :)), but the `label` and the `input` should definitely be in the _same_ component, because one should not be present without the other.